bin/grouper-fe

```diff
@@ -66,6 +66,8 @@ def main(argv):
     parser.add_argument("-V", "--version", action="version",
                         version="%%(prog)s %s" % grouper.__version__,
                         help="Display version information.")
+    parser.add_argument("-a", "--address", type=str, default="",
```

Comment:
The method to my madness here is that the default in the config file is `""`, so I made this match.  But `None` would be better.  Is the `null` value supported in the YAML that we use, so that I can document that people can set `address: null`?  Or am I just making this all too complicated?